### DISCUSSION TITLE
Moving between panes in locked mode.

### DISCUSSION THREAD

**Discussion by RizaHKhan** [zellij-org/zellij#3091] [Q&A]
Hello, 

Been using Zellij as my daily driver for a couple of months now with pretty much the default config. 

The first thing that would be amazing is moving between panes in locked mode. Much like how [tmux](https://github.com/aserowy/tmux.nvim) allows. 

Is this possible will Zellij?

**Comment by robrecord** [comment]
Yes, you can do this. In the config, look at the keybinds under `shared_except "locked" {`. Move the keybinds that say `MoveFocusOrTab` and `MoveFocus` to a new section that says simply `shared {`. Make sure to add a closing `{` after the keybinds you moved.
